core/iwasm/common/wasm_c_api.c

```diff
@@ -299,6 +299,11 @@ wasm_config_new(void)
 
     memset(config, 0, sizeof(wasm_config_t));
     config->mem_alloc_type = Alloc_With_System_Allocator;
+#if defined(os_writegsbase)
+    /* enable segue for all load/store operations */
+    config->segue_flags = 0x1F1F;
```

Comment:
i feel it's safer to disable segue stuff by default.
https://github.com/bytecodealliance/wasm-micro-runtime/issues/2736
how do you think?

Reply:
OK, maybe some targets don't support modifying GS register during execution, let's disable it by default.

Reply:
Done.